packages/perseus-score/src/widgets/numeric-input/score-numeric-input.ts

```diff
@@ -95,6 +98,8 @@ function scoreNumericInput(
             inexact: true, // TODO(merlob) backfill / delete
             maxError: answer.maxError,
             forms: validatorForms,
+            // Pass locale-specific decimal separator like expression scorer does
+            ...(locale && {decimal_separator: getDecimalSeparator(locale)}),
```

Comment:
We'd need to add this to expression and possibly input number as well.

Reply:
Expression already had logic for this, as it uses KAS. I've added it to Input Number, however.